### ISSUE TITLE
Навести порядок в скриптах

### ISSUE THREAD

**Issue by umputun** [radio-t/radio-t-site#1]
Скрипты в `publish` хорошо бы привести в человеческий вид. В идеале хотелось бы от всего этого зоопарка избавится, как и от мелких `.py` и заменить их чем-то одним и общим, делающим тоже самое, но аккуратно и красиво.

**Comment by umputun** [comment]
добавил начальную версию publisher app. Досконально не тестировал, но на вид все должно быть в порядке.

@q210 – план в том, чтоб все запускалось через нее, т.е. она должна быть основным entrypoint

От этого, есть несколько просьб к питоновой части (и не только):

- перенести все, что относится к тагам в свой подкаталог, `publiser/mp3tags`
- модифицировать Makefile (поменать вызовы в контейнер для всего, что про тэги на прямые вызовы py скриптов)
- ту часть, что invoke видимо не надо? это будет часть конфигурации внутри mp3tags

**Comment by q210** [comment]
ок, постараюсь за сегодня сделать

**Comment by q210** [comment]
сделал в [PR#129](https://github.com/radio-t/radio-t-site/pull/129)

**Comment by umputun** [comment]
мне кажется, что логика с `$EPISODE` в Makefile  основана на предположении что этот EPISODE где-то доступен (или передается)? Если это так, то передача EPISODE это аварийный вариант, например когда подкаст уже опубликован, но по какой-то причине мы хотим повторить один из шагов.

В обычном сценарии внутри publisher он берет этот эпизод автоматически. От этого возникает вопрос/проблема - как это передавать upload-mp3 и print-mp3-tags

**Comment by q210** [comment]
да, сейчас предполагается что $EPISODE передаётся каждый раз - это то, как работает скрипт upload.sh в мастере, makefile эту логику повторяет для этих команд.

Если надо, то сделать параметр опциональным - думаю не проблема, вынесу https://github.com/radio-t/radio-t-site/blob/publisher-go/publisher/app/main.go#L80 как самостоятельную команду и буду дергать ее, если $EPISODE не передан
